src/Buttons/__tests__/BorderlessButton.test.tsx

```diff
@@ -65,4 +71,46 @@ describe('Borderless Button', function () {
       assert.strictEqual(clickSpy.callCount, 0);
     });
   });
+  context('when forwardRef prop is present', function () {
+    beforeEach(function () {
+      const ButtonRefExample = () => {
+        const ref = useRef<HTMLInputElement>(null);
+        const onButtonClick = () => {
+          setTimeout(() => {
+            if (ref.current) {
+              ref.current.focus();
+            }
+          });
```

Comment:
Do you still need the `setTimeout` in the test?

Reply:
Also removed through 644e818, thanks!